services/src/modules/resource-repository/s3.ts

```diff
@@ -184,6 +184,7 @@ export class S3ResourceRepository implements ResourceRepository {
   }
 }
 
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
 function isAwsError(e: any): e is AWS.AWSError {
```

Comment:
Since this function receives the value of the `catch` clause, even if we don't know the exact type of the error we do know it is derived from the `Error` class (all other error types should inherit from it).
We can use `Error` instead of `any` here

Reply:
Not really. In function body there are checks on `e.something` properties that aren't Error's properties. So we can or to use any or to create new variable with casting. In this case any is better IMO. Also all user-defined type guard examples are with any.